packages/dds/cell/src/cell.ts

```diff
@@ -43,6 +44,58 @@ interface ICellValue {
     value: unknown;
 }
 
+interface ICellContent {
+    /**
+     * The actual value contained in the `Cell`, which needs to be wrapped to handle `undefined`.
+     */
+    value: unknown;
+    /**
+     * The attribution key contained in the `Cell`.
+     * @alpha
+     */
+    attribution: unknown;
```

Comment:
might as well type this as an `AttributionKey`:
```suggestion
    attribution?: AttributionKey;
```